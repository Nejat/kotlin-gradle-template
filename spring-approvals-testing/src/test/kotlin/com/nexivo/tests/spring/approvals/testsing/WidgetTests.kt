package com.nexivo.tests.spring.approvals.testsing

import com.nexivo.spring.approvals.testing.Widget
import org.approvaltests.Approvals
import org.approvaltests.reporters.DiffReporter
import org.approvaltests.reporters.UseReporter
import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.test.context.junit4.SpringRunner

@RunWith(SpringRunner::class)
@UseReporter(DiffReporter::class)
class WidgetTests {

    @Test
    fun widgets_tested () {

        val sut = Widget(21.0)

        Approvals.verify(sut)
    }
}