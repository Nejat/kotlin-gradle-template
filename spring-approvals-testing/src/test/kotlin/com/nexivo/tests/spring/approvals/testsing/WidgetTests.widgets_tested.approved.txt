21.0, 42.0, 10.5